From 88fe2e2982525d577df8a8ec69c5cc222a592fb9 Mon Sep 17 00:00:00 2001
From: Pavel <sol.pablo2015@yandex.ru>
Date: Sun, 23 Mar 2025 21:59:11 +0300
Subject: [PATCH 2/3] Fix comments

---
 20250317/1/server.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/20250317/1/server.py b/20250317/1/server.py
index 2f54164..95b2e15 100644
--- a/20250317/1/server.py
+++ b/20250317/1/server.py
@@ -99,7 +99,7 @@ def handle_command(command):
     elif cmd == "attack":
         try:
             monster_name = parts[1]
-            weapon_name = "sword"  # Default weapon
+            weapon_name = "sword"
             if len(parts) > 3 and parts[2] == "with":
                 weapon_name = parts[3]
 
@@ -138,7 +138,6 @@ def start_server():
         client_socket, addr = server_socket.accept()
         print(f"Client connected: {addr}")
 
-        # Send welcome message
         client_socket.send("<<< Welcome to Python-MUD 0.1 >>>\n".encode())
 
         while True:
-- 
2.47.1

